src/index.js

```diff
@@ -9,4 +9,8 @@ export { default as getStoredState } from './getStoredState'
 export { default as createPersistoid } from './createPersistoid'
 export { default as purgeStoredState } from './purgeStoredState'
 
+export { default as autoMergeLevel1 } from './stateReconciler/autoMergeLevel1'
```

Comment:
from what I understand it's what we have established in the other issue, correct me if im wrong

Reply:
👍 I think so, but definitely want to get other opinions here. Are there any cases where we cannot rely on code splitting? Will this negatively impact any use cases?

Reply:
dont think so, I consider this new setup with code-splitting the most "optimized"